README.md

```diff
@@ -20,6 +20,7 @@ Language Translations:
 * [简体中文 (zh_cn)](translations/README_zh_cn.md)
 * [繁體中文 (zh_tw)](translations/README_zh_tw.md)
 * [日本語 (ja_JP)](translations/README_ja_JP.md)
+* [French (fr_FR)](translations/README_fr_FR.md)
```

Comment:
The convention thus far has been to use the name of the language in the target language which I believe would be Francais in this case?

Reply:
I completely forgot. I'll patch it immediatly.